src/api/schemas.py

```diff
@@ -50,14 +67,14 @@ class CategoryResponse(ResponseBase):
 class TaskRequest(RequestBase):
     """Класс модели запроса для Task."""
 
-    id: NonNegativeInt = Field(...)
-    title: StrictStr = Field(...)
-    name_organization: StrictStr = Field(...)
-    deadline: date = Field(..., format=DATE_FORMAT)
-    category_id: NonNegativeInt = Field(...)
-    bonus: NonNegativeInt = Field(...)
-    location: StrictStr = Field(...)
-    link: StrictStr = Field(...)
+    id: NonNegativeInt = Field(..., description=TASK_ID_DESCRIPTION)
```

Comment:
Гораздо проще будет читаться если не использовать константы: 
```suggestion
    id: NonNegativeInt = Field(..., description="Уникальный идентификатор задачи")
```

Reply:
Все константы из дескрипшена убрать?